.changeset/slimy-rules-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"kubernetes-agent": minor
```

Comment:
Kinda devils advocate, but should this be a `major` bump?

Reply:
It's backwards compatible, but adding new functionality, so left it as a `minor` according to SemVer.

Reply:
yep, happy with that. Just wanted to ask the question :)